source/Calamari.Tests/Fixtures/Bash/BashFixture.cs

```diff
@@ -58,6 +59,19 @@ public void ShouldConsumeParametersWithQuotes()
             output.AssertSuccess();
             output.AssertOutput("Parameters Para meter0Para meter1");
         }
+        
+        [Test]
+        [RequiresBashDotExeIfOnWindows]
+        public void ShouldNotHaveDecryptionKeyInScopeOfUserScript()
```

Comment:
Maybe a little context about why this test is actually testing we don't get the decryption key in the out when no parameters are available.

Reply:
✅ Simplified test and made it more generic/clearer in intent